src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -152,58 +143,84 @@ public IndexableRecord convertRecord(
           );
           break;
         case FAIL:
-          throw new DataException(String.format(
-              "Sink record with key of %s and null value encountered for topic/partition/offset "
-              + "%s/%s/%s (to ignore future records like this change the configuration property "
-              + "'%s' from '%s' to '%s')",
-              record.key(),
-              record.topic(),
-              record.kafkaPartition(),
-              record.kafkaOffset(),
-              ElasticsearchSinkConnectorConfig.BEHAVIOR_ON_NULL_VALUES_CONFIG,
-              BehaviorOnNullValues.FAIL,
-              BehaviorOnNullValues.IGNORE
-          ));
+          throw new DataException(
+              String.format(
+                  "Sink record with key of %s and null value encountered for topic/partition/offset"
+                      + " %s/%s/%s (to ignore future records like this change the configuration"
+                      + " property '%s' from '%s' to '%s')",
+                  record.key(),
+                  record.topic(),
+                  record.kafkaPartition(),
+                  record.kafkaOffset(),
+                  ElasticsearchSinkConnectorConfig.BEHAVIOR_ON_NULL_VALUES_CONFIG,
+                  BehaviorOnNullValues.FAIL,
+                  BehaviorOnNullValues.IGNORE
+              )
+          );
         default:
-          throw new RuntimeException(String.format(
-              "Unknown value for %s enum: %s",
-              BehaviorOnNullValues.class.getSimpleName(),
-              behaviorOnNullValues
-          ));
+          throw new ConnectException(
```

Comment:
nit: do we even need this section? Since `behaviorOnNullValues()` returns enum type `BehaviorOnNullValues`?